[2024-08-10T07:29:32.610+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-10T07:29:32.621+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETLWeatherPostgresAndPrint.load manual__2024-08-10T07:29:28.592270+00:00 [queued]>
[2024-08-10T07:29:32.626+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETLWeatherPostgresAndPrint.load manual__2024-08-10T07:29:28.592270+00:00 [queued]>
[2024-08-10T07:29:32.626+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-10T07:29:32.633+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2024-08-10 07:29:28.592270+00:00
[2024-08-10T07:29:32.638+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=2073) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-10T07:29:32.639+0000] {standard_task_runner.py:64} INFO - Started process 2077 to run task
[2024-08-10T07:29:32.639+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETLWeatherPostgresAndPrint', 'load', 'manual__2024-08-10T07:29:28.592270+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/03_ETLWeatherPostgresAndPrint.py', '--cfg-path', '/tmp/tmp895u520h']
[2024-08-10T07:29:32.640+0000] {standard_task_runner.py:91} INFO - Job 27: Subtask load
[2024-08-10T07:29:32.664+0000] {task_command.py:426} INFO - Running <TaskInstance: ETLWeatherPostgresAndPrint.load manual__2024-08-10T07:29:28.592270+00:00 [running]> on host 101007b4e0e8
[2024-08-10T07:29:32.703+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETLWeatherPostgresAndPrint' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-08-10T07:29:28.592270+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-10T07:29:28.592270+00:00'
[2024-08-10T07:29:32.704+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-10T07:29:32.706+0000] {logging_mixin.py:188} INFO - 1 Record inserted successfully into table
[2024-08-10T07:29:32.707+0000] {logging_mixin.py:188} INFO - PostgreSQL Connection Closed
[2024-08-10T07:29:32.707+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-10T07:29:32.707+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-10T07:29:32.711+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETLWeatherPostgresAndPrint, task_id=load, run_id=manual__2024-08-10T07:29:28.592270+00:00, execution_date=20240810T072928, start_date=20240810T072932, end_date=20240810T072932
[2024-08-10T07:29:32.741+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-10T07:29:32.748+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-10T07:29:32.749+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
